pkg/types/ccipocr3/chain_accessor.go

```diff
@@ -36,7 +36,7 @@ type AllAccessors interface {
 	// GetContractAddress returns the contract address that is registered for the provided contract name and chain.
 	// WARNING: This function will fail if the oracle does not support the requested chain.
 	//
-	// Deprecated: use Metadata() instead.
+	// TODO(NONEVM-1865): do we want to mark this as deprecated in favor of Metadata()?
```

Comment:
Seems like a good approach, lets decide when we know more about what it's like to use CAL. Maybe "GetContractAddress" is the only metadata we'll ever care about.